test/fixtures/e2eapi.go

```diff
@@ -71,7 +71,7 @@ func InvokeE2EAPIPOST(format string, body string, args ...interface{}) string {
 		if err == nil && resp.StatusCode < 300 {
 			return true, nil
 		}
-		fmt.Printf("Got error %v, response %v, retrying.\n", err, *resp)
+		fmt.Printf("Got error %v, response %v, retrying.\n", err, resp)
```

Comment:
With this change, will we be printing the address of the pointer instead of the object content?

Reply:
fixed